index.html

```diff
@@ -66,6 +68,128 @@ <h2 class="role">I'm a Software Developer</h2>
+                    </p>
+                    <ul class="tag">
+                        <li class="tag-description-2">HTML</li>
+                        <li class="tag-description-2">Bootstrap</li>
+                        <li class="tag-description-2">Ruby</li>
+                    </ul>
+                </div>
+                <button type="button" class="see-prj inter700 wide">See project</button>
+            </div>
+            <div class="card-flex proArt">
+                <div class="text-container">
+                    <h3 class="card-title crete font-white">Profesional Art Printing Data</h3>
+                    <p class="card-description font-white">
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required. Has been the industry's standard.
+                    </p>
+                    <ul class="tag">
+                        <li class="tag-description-2">HTML</li>
+                        <li class="tag-description-2">Bootstrap</li>
+                        <li class="tag-description-2">Ruby</li>
+                    </ul>
+                </div>
+                <button type="button" class="see-prj inter700 wide">See project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <h2 class="about-title crete">About me</h2>
+        <p class="about-description inter400">
+            Hello I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <button type="button" class="see-prj inter700">Get my resume</button>
+
+        <div class="div-line"></div>
+
+        <div class="skills">
+            <div class="skills-title">
+                <img src="./icons/Rectangle 55.png" alt="" />
+                <h2 class="skills-type inter700">Language</h2>
+            </div>
+            <ul class="skills-list">
+                <li class="list-item inter400">JavaScript</li>
+                <li class="list-item inter400">Ruby</li>
+                <li class="list-item inter400">HTML</li>
+                <li class="list-item inter400">CSS</li>
+            </ul>
+
+            <div class="skills-title">
+                <img src="./icons/Rectangle 56.png" alt="" />
+                <h2 class="skills-type inter700">Frameworks</h2>
+            </div>
+            <ul class="skills-list">
+                <li class="list-item inter400">React.js</li>
+                <li class="list-item inter400">Ruby on Rails</li>
+                <li class="list-item inter400">RSpec</li>
+                <li class="list-item inter400">Capybara</li>
+            </ul>
+
+            <div class="skills-title">
+                <img src="./icons/Rectangle 57.png" alt="" />
+                <h2 class="skills-type inter700">Skills</h2>
```

Comment:
- [ ] Nice work on the "about me" section, however, according to the project's [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) the categories "Languages", "Frameworks" etc should be crapped inside an `<li>` tag as well 🔽 

  > Please use ul and `li` for categories in "About myself" section. `Languages`, `Frameworks`, and `Skills` should be list items.

Reply:
Thank you for your advice, the change it's done.